web/impact/impact/v1/views/allocate_applications_view.py

```diff
@@ -37,6 +37,7 @@
 
 
 class AllocateApplicationsView(ImpactView):
+    '''Endpoint for allocation of applications to a judge.'''
```

Comment:
it might be useful to add some context as to how this is triggered (via the button on the judge dashboard etc)

Reply:
This information is in the documentation. (https://github.com/masschallenge/standards/pull/102)

I would not want to see documentation suggesting that we've got the dependency flow the wrong way around - this service should not know about where it's used, and it should not need to be updated if some new page makes use of it. 

Pushing back on this one.